doc/going-further/http-request.md

```diff
@@ -0,0 +1,65 @@
+# HTTP requests
+
+## The `request()` function
+
+The `request()` function allows to make HTTP requests easily. It performs HTTP
+request and returns an instance of
+`Symfony\Contracts\HttpClient\ResponseInterface`:
+
+```php
+use Castor\Attribute\AsTask;
+
+use function Castor\request;
+
+#[AsTask]
+function foo()
+{
+    echo request('GET', 'https://example.org')->getContent(), \PHP_EOL;
+}
+```
+
+## The `http_client()` function
+
+If you need to have a full control on the HTTP client, you can access the
+`HttpClientInterface` directly with the `http_client()` function:
+
+```php
+use Castor\Attribute\AsTask;
+
+use function Castor\http_client;
+
+#[AsTask]
+function foo()
+{
+    $client = http_client()
+        ->withOptions([
+            'verify_peer' => false,
+            'timeout' => 10,
+        ])
+    ;
+}
+```
+
+## Configuring the HTTP client globally
+
+If you need to configure the HTTP client globally, you can do it in the context
+creator:
+
+```php
+use Castor\Attribute\AsContext;
+use Castor\GlobalContext;
+use Castor\Context;
+
+#[AsContext(name: 'preprod')]
+function preprodContext(): Context
+{
+    // $client = ...
+    GlobalContext::setHttpClient($client)
```

Comment:
here too, this does not exist anymore. IMHO we should remove this chapter.

I'll do another PR to explain how to extends Castor default... one day